client/data/courses/use-course-data.ts

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react';
+import useCourseQuery from './use-course-query';
+import type { Course, VideoSlug } from './types';
+
+interface Result {
+	course?: Course;
+	videoSlugs: VideoSlug[];
+	completedVideoSlugs: VideoSlug[];
+	isCourseComplete: boolean;
+}
+
+const useCourseData = ( courseSlug: string ): Result => {
+	const { data } = useCourseQuery( courseSlug, { retry: false } );
```

Comment:
I see `retry = false` was copied over from the old call to `useCourseQuery()`, but I wonder if it's still needed? Maybe it was trying to work around the fact the hook re-fetch all the time, but now that you've specified a `staleTime` perhaps it's better to just leave the retry behaviour as default.

Reply:
I think retry is for re-fetch when the query is failed but I'm not sure why we disabled it before. I agree with leaving it as default is better except we have a special reason for it.

Reply:
you are both right! I've added a PR to remove the `retry: false`: https://github.com/Automattic/wp-calypso/pull/59500

Reply:
Removed retry on https://github.com/Automattic/wp-calypso/pull/59500